app/routes/articles/components/ArticleDetail.js

```diff
@@ -19,7 +19,8 @@ type Props = {
   comments: Array<CommentEntity>,
   loggedIn: boolean,
   author: UserEntity,
-  currentUser: UserEntity
+  currentUser: UserEntity,
+  youtubeParams: Object
```

Comment:
youtubeParams should be optional, and maybe it should be a part of the ArticleEntity instead?

Reply:
And Type YoutubeParams